site-generator/pom.xml

```diff
@@ -36,6 +36,7 @@
     <properties>
         <versions.txt.dir>${project.parent.basedir}/models/standalone</versions.txt.dir>
         <output.dir>${project.basedir}/target/wildscribe-generated</output.dir>
+        <PRODUCT_VERSION>JBoss-EAP-7.3.0.CD16</PRODUCT_VERSION>
```

Comment:
Given this is a community project I think we should use something like `container.version` or `server.version` with no default value.